nodes/config/ui_base.js

```diff
@@ -761,6 +762,7 @@ module.exports = function (RED) {
          * @param {*} widget
          */
         node.register = function (page, group, widgetNode, widgetConfig, widgetEvents) {
+            // console.log('dashboard 2.0, UIBaseNode: node.register(...)', page, group, widgetNode, widgetConfig, widgetEvents)
```

Comment:
commented console.log

Reply:
resolved in [1b74cbf](https://github.com/FlowFuse/node-red-dashboard/pull/634/commits/1b74cbfddc0e19c215a6d4a4b28f6af3c0860ed8)